src/Compilers/CSharp/Test/Symbol/Symbols/Source/EventTests.cs

```diff
@@ -8,6 +8,8 @@
 using Microsoft.CodeAnalysis.Test.Utilities;
 using Roslyn.Test.Utilities;
 using Xunit;
+using System.IO;
+using Microsoft.CodeAnalysis.Emit;
```

Comment:
They are for line 412. But they will go away with Chuck's suggestion to use EmitToImageReference instead.

```
using (var output = new MemoryStream())
            {
                libComp.Emit(output, options: new EmitOptions(metadataOnly: true));
```